src/main/java/io/projectriff/processor/Processor.java

```diff
@@ -321,47 +314,49 @@ public void run() {
     }
 
     /**
-     * This converts an RPC representation of an {@link OutputFrame} to an at-rest {@link Message}, and creates a publish request for it.
+     * This converts an RPC representation of an {@link OutputFrame} to an at-rest CloudEvent, and creates a publish request for it.
      */
     private PublishRequest createPublishRequest(OutputFrame next, String topic) {
-        Message msg = Message.newBuilder()
-                .setPayload(next.getPayload())
-                .setContentType(next.getContentType())
-                .putAllHeaders(next.getHeadersMap())
-                .build();
-
         return PublishRequest.newBuilder()
-                .setValue(msg.toByteString())
+                .setLiiklusEvent(
+                        LiiklusEvent.newBuilder()
+                                .setData(next.getPayload())
+                                .setDataContentType(next.getContentType())
+                                .setType("riff-event") // TODO
+                                .setSource(this.group) // TODO
```

Comment:
TODO for another PR or sth to figure out now?

Reply:
Later, needs to figure out even the meaning we want to give this.

The whole PR is ready for review as-is, but what you see is what you get